arch/platform/native/platform.c

```diff
@@ -238,7 +247,18 @@ platform_init_stage_one()
   gpio_hal_init();
   button_hal_init();
   leds_init();
-  return;
+#if defined(__linux__) || defined(__MACH__)
+  struct csprng_seed seed;
+#ifdef __linux__
+  if(getrandom(seed.u8, CSPRNG_SEED_LEN, GRND_RANDOM) != CSPRNG_SEED_LEN) {
+#else /* __linux__ */
+  if(SecRandomCopyBytes(kSecRandomDefault, CSPRNG_SEED_LEN, seed.u8)) {
+#endif /* __linux__ */
+    assert(false);
```

Comment:
Can this state be recovered, or is it a fatal error? If it is fatal, it sounds like this should be an abort/exit.

That would also allow eliminating the else-block and just call `csprng_feed` without indentation.

Reply:
It is not that fatal as long as the return value of `csprng_rand` is always being checked. I will remove that assert.